### ISSUE TITLE
Document how to port static Web apps

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-docs#151]
@frankrousseau, do you think it would be worth adding to the doc [my article](http://esquisses.clochix.net/2014/10/16/cozy-static-en/) on how to port a static Web app to Cozy ?

**Comment by frankrousseau** [comment]
Yes it would be good. But I don't understand the "Bonus: Configure your application for Cozy" part.
